content/desktop/use-desktop/container.md

```diff
@@ -29,46 +29,92 @@ When you [inspect a container](#inspect-a-container), the **Stats** tab displays
-    - Select and view external links that may be in your logs. 
+Select **Logs** to see logs from the container. You can also:
+
+- Use `Cmd + f`/`Ctrl + f` to open the search bar and find specific entries.
+  Search matches are highlighted in yellow.
+- Press `Enter` or `Shift + Enter` to jump to the next or previous search match
+  respectively. 
+- Use the **Copy** icon in the top right-hand corner to copy all the logs to
+  your clipboard.
+- Automatically copy any logs content by highlighting a few lines or a section
+  of the logs.
+- Use the **Clear terminal** icon in the top right-hand corner to clear the
+  logs terminal. 
+- Select and view external links that may be in your logs. 
 
 ### Inspect
 
 Select **Inspect** to view low-level information about the container. It displays the local path, version number of the image, SHA-256, port mapping, and other details.
 
 ### Integrated terminal
 
-From the **Exec** tab, lets you use the integrated terminal, on a running container, directly within Docker Desktop. You are able to quickly run commands within your container so you can understand its current state or debug when something goes wrong.
+From the **Exec** tab, you can use the integrated terminal, on a running
+container, directly within Docker Desktop. You are able to quickly run commands
+within your container so you can understand its current state or debug when
+something goes wrong.
+
+Using the integrated terminal is the same as running one of the following commands:
+
+- `docker exec -it <container-id> /bin/sh`
+- `docker exec -it <container-id> cmd.exe` when accessing Windows containers
+- `docker debug <container-id>` when using debug mode
 
-Using the integrated terminal is the same as running `docker exec -it <container-id> /bin/sh`, or `docker exec -it <container-id> cmd.exe` if you are using Windows containers, in your external terminal. It also:
+The integrated terminal:
 
-- Automatically detects the default user for a running container from the image's Dockerfile. If no user is specified it defaults to `root`.
-- Persists your session if you navigate to another part of the Docker Dashboard and then return.
+- Persists your session and **Debug mode** setting if you navigate to another
+  part of the Docker Dashboard and then return.
 - Supports copy, paste, search, and clearing your session.
+- When not using debug mode, it automatically detects the default user for a
+  running container from the image's Dockerfile. If no user is specified, or
+  you're using debug mode, it defaults to `root`.
+
+#### Open the integrated terminal
 
 To open the integrated terminal, either:
-- Hover over your running container and select the **Show container actions** menu. From the drop-down menu, select **Open in terminal**.
+
+- Hover over your running container and select the **Show container actions**
+  menu. From the drop-down menu, select **Open in terminal**.
 - Select the container and then select the **Exec** tab.
```

Comment:
I _think_ there's two ways to open the integrated terminal. 

1. Would be clicking the name of the container, which would automatically send user's to the logs tab where they can then click the exec tab (if docker debug is not enabled). 
2. Would be clicking the menu action on a container (the 3 dots) and clicking the `open in terminal` option. When clicking the `open in terminal` option, it should automatically send users to the exec tab. 

I only bring this up because of the step where it says to select the Exec tab. If it's the first way to get to the integrated terminal it'd make sense to keep but if we are going with the second way in going to the integrated terminal, it'd already send users to that tab so they don't need to have that step included.

Reply:
I think it is correct, although easy to misinterpret. That unordered list is supposed to be like, do either or. I think it can be misinterpreted as an ordered list without `or` between the choices. I added `Or,` before the second item.

Reply:
Ah that makes total sense now! I didn't realize! 🙇🏼  thank you for clarifying it for me!!